dispatch/BESContextManager.cc

```diff
@@ -213,8 +211,9 @@ void BESContextManager::dump(ostream &strm) const
 BESContextManager *
 BESContextManager::TheManager()
 {
-    std::call_once(d_euc_init_once,BESContextManager::initialize_instance);
-    return d_instance;
+    static BESContextManager manager;
+    // std::call_once(d_euc_init_once,BESContextManager::initialize_instance);
```

Comment:
Please remove the commented out code and also delete the d_euc_init_once variable.

Reply:
10-4